docs/src/changelog/index.md

```diff
@@ -7,6 +7,7 @@ Versions and changes-within-versions are listed in reverse-chronological order:
 
 Version 0.1.8 is the current Reach release candidate version.
 
++ 2022/02/28: Added `{!js} url`, `{!js} metadataHash`, and `{!js} clawback` options, as well as frontend documentation, for token minting.
```

Comment:
No. You just say "Added launchToken, for token minting in frontends". The feature didn't exist at all before